apps/mobile/src/libs/components/checkbox-categories-form/libs/helpers/get-categories-sorted-by-score/get-categories-sorted-by-score.helper.ts

```diff
@@ -0,0 +1,19 @@
+import { type QuizScoresGetAllItemResponseDto } from "~/libs/types/types";
+
+type GetCategoriesSortedByScoreArguments = {
+	categories: QuizScoresGetAllItemResponseDto[];
+	useDescendingOrder?: boolean;
```

Comment:
```suggestion
	order?: "ASC" | "DESC"
```
And give a default value